dir.props

```diff
@@ -332,6 +332,7 @@
   <!-- Change define constants as needed -->
   <PropertyGroup Condition="'$(MsbuildDebugger)' == 'true' AND '$(MonoBuild)' != 'true'">
     <DefineConstants>$(DefineConstants);FEATURE_MSBUILD_DEBUGGER</DefineConstants>
+    <DefineConstants Condition="$(LocalizedBuild) == 'true'">$(DefineConstants);LOCALIZED_BUILD</DefineConstants>
```

Comment:
I think the `Localized_Build` define needs to be taken out. During release builds, I doubt that `MSBuildDebugger` is set to true. Or is it? What's that anyway?